pg_rman.h

```diff
@@ -259,6 +259,7 @@ extern bool data_checksum_enabled;
 extern int do_backup(pgBackupOption bkupopt);
 extern BackupMode parse_backup_mode(const char *value, int elevel);
 extern bool fileExists(const char *path);
+extern bool get_standby_signal_filepath(char *path, size_t size);
```

Comment:
The name of the current function feels like it is checking if the XXX.signal file exists.
However, the XXX.signal file has been newly added from pg12 or higher, and it does not have a the XXX.signal file pg11 or under versions, and the XXX.singal file is may change the name of the file in future PG versions (not currently planned, but just in case).

So, how about changing the name of the function to be used in all branches?

Of course, the branch PG11 and under version do not use the function in the restore.c, but it is used in the backup.c.

And come to think of it, if need to the back-patch, It needs to modify it a bit in the branch PG11 and under version..
In this case, I will correct it and proceed with the back-pach.

Reply:
Oh... I'm sorry.
Looking at the contents of the PR you suggested, it was necessary for the branch of PG12 or higher versions.
So you can ignore this comment.
However, if you think that it is better to apply only to backup.c in the branch PG11 and under versions, or if you think that the generically usable function name is good, please responder to this comment as well.

Reply:
Thanks, I didn't notice that. But, sorry I didn't come up with a better name.
The postgresql source codes have "readRecoverySignalFile()", so I think the above name is not bad.